app/src/main/java/com/andrew/tasky/agenda/data/event/EventRepositoryImpl.kt

```diff
@@ -308,6 +311,22 @@ class EventRepositoryImpl @Inject constructor(
         }
     }
 
+    private fun scheduleNotification(event: Event) {
+        scheduler.schedule(
+            agendaId = event.id,
+            time = ReminderTimeConversion.toZonedEpochMilli(
+                startLocalDateTime = event.startDateAndTime,
+                reminderTime = event.reminderTime
+            )
+        )
+    }
+
+    private suspend fun cancelScheduledNotification(eventId: String) {
+        db.getEventDao().getEventById(eventId)?.toEvent(this)?.let {
```

Comment:
Should be fine to just call `cancel` with the passed ID here. If there's no scheduled job it will simply ignore it